charts/opsgenie-heartbeat/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "opsgenie-heartbeat.fullname" . }}
+  labels:
+    {{- include "opsgenie-heartbeat.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "opsgenie-heartbeat.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "opsgenie-heartbeat.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "opsgenie-heartbeat.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          {{- if .Values.existingSecret }}
+          - name: OPSGENIE_APIKEY
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.existingSecret }}
+                key: "apikey"
+          {{- end }}
+          - name: OPSGENIE_HEARTBEAT_NAME
```

Comment:
Let's add possibility here too to pass OPSGENIE_LIVE_RETRY var (in case if somebody will need to change this param from 1h to more)

Reply:
Done